bb-new.html

```diff
@@ -0,0 +1,296 @@
+              <li class="nav-list-item"><a href="#">Healthcare</a></li>
+              <li class="nav-list-item"><a href="#">Professional Services</a></li>
+              <li class="nav-list-item"><a href="#">Education</a></li>
+              <li class="nav-list-item"><a href="#">Government</a></li>
+              <li class="nav-list-item"><a href="#">Utilities & Manufactoring/a></li>
+              <li class="nav-list-item"><a href="#">Transportation</a></li>
+              <li class="nav-list-item"><a href="#">Consumer industries</a></li>
+            </ul>
+          </div>    
+        </div>
+        <div class="col">
+          <a href="#partners">Partners</a>
+          <div class="dropwdown">
+            <ul>
+              <li class="nav-list-item"><a href="#">Solution Partners</a></li>
+              <li class="nav-list-item"><a href="#">IVS Partners</a></li>
+              <li class="nav-list-item"><a href="#">Cloud Partners</a></li>
+              <li class="nav-list-item"><a href="#">OEM Partners</a></li>
+              <li class="nav-list-item"><a href="#">Developers</a></li>
+              <li class="nav-list-item"><a href="#">Authroized Training Partners</a></li>
+            </ul>
+          </div>    
+        </div>
+        <div class="col">
+          <a href="#support">Support</a>
+          <div class="dropwdown">
+            <ul>
+              <li class="nav-list-item"><a href="#">Support by Product</a></li>
+              <li class="nav-list-item"><a href="#">Smart Phone & Apps</a></li>
+              <li class="nav-list-item"><a href="#">Product Information</a></li>
+            </ul>
+          </div>    
+        </div>
+        <div class="col">
+          <a href="#blog">Blog</a>
+          <div class="dropwdown">
+            <ul>
+              <li class="nav-list-item"><a href="#">Enterprise Blog</a></li>
+              <li class="nav-list-item"><a href="#">Developer Blog</a></li>
+              <li class="nav-list-item"><a href="#">Support Blog</a></li>
+            </ul>
+          </div>    
+        </div>
+        <div class="col">
+          <a href="#blog">Seach icon</a>
+        </div>
+        <div class="col">
+          <a href="#headphone">Headphone icon</a>  
+          <div class="dropwdown">
+            <ul>
+              <li class="nav-list-item"><a href="#">Register Products</a></li>
+              <li class="nav-list-item"><a href="#">Contact Sales</a></li>
+              <li class="nav-list-item"><a href="#">Contact Us</a></li>
+            </ul>
+          </div>    
+        </div>                 
+        </nav>                 
+            <div class="row">
+              <div class="col">
+                <h1>BlackBerry 2021 Threat Report Now Available</h1>
+                <p>Learn about the most significant cybersecurity events of 2020 so you can prepare for the year ahead</p>
+                <button>READ THE REPORT</button>
+              </div>
+            </div>
```

Comment:
How is this part of the `<header>`?

Reply:
When looking into what goes into header, I did an”inspect” on BlackBerry page and this along with navigation was in included in the header tag.